Tests/TuistGeneratorTests/Generator/ConfigGeneratorTests.swift

```diff
@@ -997,6 +997,44 @@ final class ConfigGeneratorTests: TuistUnitTestCase {
         XCTAssertEqual(targetSettingsResult, "YES")
     }
 
+    func test_generateTargetConfig_when_defaultSettingsIsRecommendedWithExcludingTEST_HOST_then_TEST_HOSTIsNil() async throws {
+        let settings = Settings.test(defaultSettings: .essential(excluding: ["TEST_HOST"]))
+        try await generateTestTargetConfigTestHostSettings(settings)
+        let targetSettingsResult = try pbxTarget
+            .buildConfigurationList?
+            .buildConfigurations
+            .first { $0.name == "Debug" }?
+            .buildSettings
+            .toSettings()["TEST_HOST"]
+        XCTAssertEqual(targetSettingsResult, nil)
```

Comment:
Can we follow the `// Given // When // Then` convention? You can take inspiration from the other test methods in this test suite 🤞